gips/data/landsat/landsat.py

```diff
@@ -1265,10 +1267,10 @@ def _readqa(self):
             qadatafile = self.assets[asset].extract([self.metadata['qafilename']])
         else:
             # Use tar.gz directly using GDAL's virtual filesystem
-            qadatafile = os.path.join('/vsitar/' + self.assets[asset].filename, self.metadata['qafilename'])
-        qaimg = gippy.GeoImage(qadatafile)
-        return qaimg
-
+            qadatafile = os.path.join('/vsitar/' + self.assets[asset].filename,
+                                      self.metadata['qafilename'])
+        npqa = gippy.GeoImage(qadatafile).read().astype('uint16')
```

Comment:
Looks like `read_raw` will handle this w/o the conversion.

Reply:
Unfortunately it looks like that's not exposed to python; `gippy.GeoImage(qadatafile).read_raw` right there caused an AttributeError when I tried it just now.

Reply:
it is  a method of `GeoRaster`, so 
```
>>> import gippy as gp
>>> i = gp.GeoImage.create()
>>> a = i[0].read_raw()
>>> a.dtype
dtype('uint8')
>>> i.type().string()
'uint8'
```

Reply:
So technically it's both; I didn't think to go look in GeoRaster when the GeoImage call failed.  I'll try again after standup.